src/comparator/message_comparator.py

```diff
@@ -139,88 +138,3 @@ def _compare_nested_enums(self, nested_enum_dict_original, nested_enum_dict_upda
                 nested_enum_dict_update[name],
                 self.finding_container,
             ).compare()
-
-    def _compare_resources(
-        self,
-        resource_original,
-        resource_update,
-    ):
-        if not resource_original and not resource_update:
-            return
-        # 1. A new resource definition is added.
-        if not resource_original and resource_update:
-            self.finding_container.addFinding(
-                category=FindingCategory.RESOURCE_DEFINITION_ADDITION,
-                proto_file_name=self.message_update.proto_file_name,
-                source_code_line=resource_update.source_code_line,
-                message=f"A message-level resource definition `{resource_update.value.type}` has been added.",
-                change_type=ChangeType.MINOR,
-            )
-            return
-        # 2. Message-level resource definitions removal may not be breaking change since
-        # the resource could be moved to file-level resource definition.
-        # 3. Note that the type change of an existing resource definition is like one resource
-        # is removed and another one is added.
```

Comment:
The resource removal and pattern changes are also checked at the file_set comparator. If the resource option in message level is removed but added back to the file-level resource definition, the database should still have that resource registered, it is not considered breaking.